guide/dynamics/dynamics-bloch-redfield.rst

```diff
@@ -255,11 +290,12 @@ As we have already discussed, the Bloch-Redfield master equation requires transf
 The time-dependent Bloch-Redfield solver in QuTiP relies on the efficient numerical computations afforded by the string-based time-dependent format, and Cython compilation.  As such, all the time-dependent terms, and noise power spectra must be expressed in the string format.  To begin, lets consider the previous example, but formatted to call the time-dependent solver:
 
 
-.. ipython::
+.. plot::
```

Comment:
> * I noticed that some blocks using `plot` have no plots. In the rendered file they appear with a `source code` at the end, which belongs to the plot directive. It breaks the flow of the text a bit. Then I realized that you probably just want to keep the same namespace. If there is no easy way around it, just ignore those comments of mine.


`plot` is used regardless of whether there needs to be a plot or not so that all the code is in the same namespace, we can work around it but don't need to, let me try to see if I can remove the `source code` thing at the end. 


> * If we are about to use `>>>` to indicate input code, it would be good to be consistent throughout the doc. I noticed that in `testcode` blocks you are writing the input without `>>>`. Is it possible also use `>>>`?

It isn't possible to use `>>>` with `testcode`. I decided between the `testcode` and `doctest` as when it was useful, i.e. `doctest` when multiple things are to be output in quick succession and `testcode` otherwise. 

> * We can also add these conventions to `directives.rst`, or other guides for doc contributors.

I'll add some of these suggestions to the doc.

Reply:
Is it okay if none of the plot directives show the link to source code?

Reply:
> Is it okay if none of the plot directives show the link to source code? There isn't an easy way to do it for individual blocks.

That would be excellent. If that is too much work you can also leave it. I get now that you want to keep the namespace.